src/PowerShellEditorServices.Engine/PowerShellEditorServices.Engine.csproj

```diff
@@ -21,4 +21,7 @@
     <PackageReference Include="System.Security.Principal.Windows" Version="4.5.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Services\CodeLens\" />
```

Comment:
What does this entry do? Is there a reason CodeLens requires it when other features/services don't?

Reply:
I think it was VS that added it. I'll delete it